sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SynchronousInstrumentAccumulator.java

```diff
@@ -14,45 +14,41 @@
 import java.util.Objects;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.locks.ReentrantLock;
-import java.util.function.Function;
 
-final class SynchronousInstrumentAccumulator<B extends AbstractBoundInstrument> {
-  private final ConcurrentHashMap<Labels, B> boundLabels;
+final class SynchronousInstrumentAccumulator {
+  private final ConcurrentHashMap<Labels, Aggregator> aggregatorLabels;
   private final ReentrantLock collectLock;
   private final InstrumentProcessor instrumentProcessor;
-  private final Function<Aggregator, B> boundFactory;
 
-  SynchronousInstrumentAccumulator(
-      InstrumentProcessor instrumentProcessor, Function<Aggregator, B> boundFactory) {
-    this.boundFactory = boundFactory;
-    boundLabels = new ConcurrentHashMap<>();
+  SynchronousInstrumentAccumulator(InstrumentProcessor instrumentProcessor) {
+    aggregatorLabels = new ConcurrentHashMap<>();
     collectLock = new ReentrantLock();
     this.instrumentProcessor = instrumentProcessor;
   }
 
-  public B bind(Labels labels) {
+  Aggregator bind(Labels labels) {
     Objects.requireNonNull(labels, "labels");
-    B binding = boundLabels.get(labels);
-    if (binding != null && binding.bind()) {
+    Aggregator aggregator = aggregatorLabels.get(labels);
+    if (aggregator != null && aggregator.acquire()) {
       // At this moment it is guaranteed that the Bound is in the map and will not be removed.
-      return binding;
+      return aggregator;
     }
 
     // Missing entry or no longer mapped, try to add a new entry.
-    binding = boundFactory.apply(instrumentProcessor.getAggregator());
+    aggregator = instrumentProcessor.getAggregator();
     while (true) {
-      B oldBound = boundLabels.putIfAbsent(labels, binding);
+      Aggregator oldBound = aggregatorLabels.putIfAbsent(labels, aggregator);
```

Comment:
tiny nit: might rename this variable to `boundAggregator`

Reply:
Done.